autoload/vundle/scripts.vim

```diff
@@ -254,4 +254,36 @@ func! s:load_scripts(bang)
   return eval(readfile(f, 'b')[0])
 endf
 
+
+" ---------------------------------------------------------------------------
+" Get the dependencies of the given bundle.
+"
+" bundle -- The bundle to check. (must be installed)
+" return -- List of bundle names.
+" ---------------------------------------------------------------------------
+func! vundle#scripts#getdeps(bundle)
+  if filereadable(a:bundle['rtpath'] . '/depends.txt')
+    return eval("['" . join(readfile(a:bundle['rtpath'] . '/depends.txt'), "','") . "']")
+  elseif filereadable(a:bundle['rtpath'] . '/addon-info.json')
+    let true = 1
```

Comment:
Other people will probably have the same question. You should mention in a comment that they're defined for use inside the eval.